docs/user_best_practices.md

```diff
@@ -0,0 +1,251 @@
+# Scarf User Guide & Best Practices
+
+## Intro
+
+This is a user guide to best practices when using Scarf.  Consider this not only a guide but also an FAQ.  
+
+Currently this guide contains:
+
+1. How to use Scarf Data in existing  sales and marketing efforts

```

Comment:
It would be amazing if you could link this table of contents to the sections below

Reply:
Will do in an update.